pkgs/top-level/all-packages.nix

```diff
@@ -18870,6 +18870,7 @@ in
   prototool = callPackage ../development/tools/prototool { };
 
   qemu_kvm = lowPrio (qemu.override { hostCpuOnly = true; });
+  qemu_full = lowPrio (qemu.override { smbdSupport = true; cephSupport = true; });
```

Comment:
I fear that we might have a feature drift by having to manage feature flags both here and in the QEMU derivation... Any ideas?

Reply:
Isn't this how it is supposed to be? I've seen similar other packages like `wine`.